demisto_sdk/commands/prepare_content/integration_script_unifier.py

```diff
@@ -286,6 +287,12 @@ def insert_script_to_yml(
         script_code = IntegrationScriptUnifier.insert_module_code(
             script_code, imports_to_names
         )
+        if pack_version := get_pack_metadata(file_path=str(package_path)).get(
```

Comment:
@ilaner is this the best place to put this functionality, or maybe under `prepare_...`?